pubspec.lock

```diff
@@ -21,35 +21,35 @@ packages:
       name: archive
```

Comment:
are updates to this file necessary? I don't see you updating the pubspec.yaml file anywhere in this PR

Reply:
I just ran `flutter packages get`.
Do I need to modify something in order to create a new PR?

Reply:
You can undo the changes to this file and try pushing again.